docs/finality-provider-operation.md

```diff
@@ -85,9 +85,9 @@ backed up frequently. Loss will lead to inability to submit transactions to the
 Babylon chain, which will in turn lead to FP jailing and halt BTC Staking reward
 accumulation.
 
-- The `keyring-` folder contains your Babylon keyring, used to submit public
+* The `keyring-` folder contains your Babylon keyring, used to submit public
 randomness and finality signatures to Babylon.
-- The `fpd.db`
+* The `fpd.db`
```

Comment:
is the default db file `fpd.db` or `finality-provider.db`? If the latter, please check throughout the page